BIDSTools/Process_BIDS_DATA.py

```diff
@@ -0,0 +1,519 @@
+                        print(f"Missing 'Subject ID' in row: {row}")
+                        continue
+
+                    exp = Experiment(**row)
+
+                    if experiment.get_attribute("Subject ID") == row['Subject ID']:
+                        if exp == experiment:
+                            its_new_experience = False
+                            break
+                        else:
+                            print("There are modifications in this experiment")
+                            input_user = input(
+                                "Press 1 to continue without changes or 2 to update the experiment: ")
+                            if input_user == '1':
+                                break
+                            else:
+                                update_file(experiment, line_number, file_path, lines_updated,
+                                            headers)
+                            its_new_experience = False
+                            break
+
+            if its_new_experience:
+                add_new_experiment_to_tsv(file_path, experiment)
+
+
+def update_file(experiment, line_number, file_path, lines, headers):
+    # Filter the experiment dictionary to keep only the fields in the header
+    exp_dict = experiment.to_dict()
+    filtered_exp_dict = {key: exp_dict.get(key, '') for key in headers}
+    print(filtered_exp_dict, " found in ", file_path)
+    lines[line_number] = filtered_exp_dict
+    print(lines, "je suis lines")
+    # Write back the file with updated content
+    with open(file_path, 'w', newline='') as f:
+        writer = csv.DictWriter(f, delimiter='\t', fieldnames=headers)
+        writer.writeheader()
+        writer.writerows(lines)
+
+
+import json
+
+
+def write_static_files(template_path, file_path):
+    """
+    Fills a static file with default values based on a JSON template.
+
+    Parameters
+    ----------
+    template_path : str
+        The path to the JSON template file. This file should contain default values for various keys.
+    file_path : str
+        The path to the output file where the default values will be written.
+
+    Returns
+    -------
+    None
+    """
+    # Read the JSON content from the template file
+    with open(template_path, 'r') as template_file:
+        template_content = json.load(template_file)
+
+    # Extract default values for each key from the template
+    primary_key = {key: template_content[key]["Default value"] for key in template_content.keys()}
+
+    # Write the default values to the output file in JSON format
+    with open(file_path, 'w') as static_file:
+        json.dump(primary_key, static_file, indent=4)
+
+    print(
+        f"Written static content from {template_path} to {file_path} with primary key: {primary_key}")
+
+
+def fill_static_files(output_dir):
+    """
+    Processes static files in the specified directory. For each file that does not have a .json or .tsv extension,
```

Comment:
please define what you call 'static files' so that it is more explicit and understandable...